src/bout++.cxx

```diff
@@ -338,13 +338,14 @@ int BoutInitialise(int &argc, char **&argv) {
       return 1;
     }
   }
-  
-  output_error.enable(verbosity>0);
-  output_warn.enable(verbosity>1);
-  output_progress.enable(verbosity>2);
-  output_info.enable(verbosity>3);
-  output_debug.enable(verbosity>4); //Only actually enabled if also compiled with DEBUG
-  
+
+  output_error.enable(verbosity > 0);
+  output_warn.enable(verbosity > 1);
+  output_progress.enable(verbosity > 2);
+  output_info.enable(verbosity > 3);
+  output_debug.enable(verbosity > 4); // Only actually enabled if also compiled with DEBUG
+  output_verbose.enable(verbosity > 5);
```

Comment:
Should this be the same level as debug (or even debug > verbose)? The choice of debug or not is configure time whilst verbose is just the verbosity?

Reply:
Yes, probably makes sense for debug to be at 5 and verbose at 4